src/mixxx.cpp

```diff
@@ -1606,8 +1606,11 @@ void MixxxMainWindow::rebootMixxxView() {
 
 bool MixxxMainWindow::eventFilter(QObject* obj, QEvent* event) {
     if (event->type() == QEvent::ToolTip) {
-        // return true for no tool tips
-        switch (m_toolTipsCfg) {
+        // always show tooltips in the preferences window
+        QWidget* actWin = QApplication::activeWindow();
```

Comment:
```suggestion
        QWidget* activeWindow = QApplication::activeWindow();
```
Please spell out variable names.